src/scripts/setup-git-config.sh

```diff
@@ -0,0 +1,3 @@
+#!/bin/bash
+git config user.email "$GIT_EMAIL"
+git config user.name "$GIT_USERNAME"
```

Comment:
Note that some of the SDKs are currently not setting these as environment variables, so we will need to add them to those.

Reply:
Thanks for bringing that up! I set them in all of them I think. Some were actually hardcoding these values in the config.yml lol